tests/test_integration_employee.py

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+from unittest.mock import patch
+
+import freezegun
+
+from mora import lora
+from . import util
+
+mock_uuid = 'f494ad89-039d-478e-91f2-a63566554bd6'
+
+
+@freezegun.freeze_time('2017-01-01', tz_offset=1)
+@patch('mora.service.employee.uuid.uuid4', new=lambda: mock_uuid)
```

Comment:
Mock `uuid.uuid4` instead — it's the same object, and if that doesn't work, someone is doing something wrong. Also, it's more concise and readable.

Reply:
I honestly think it makes more sense to mock out the specific use of `uuid`, rather than mocking every use of it everywhere.